spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/SpringDocAutoConfiguration.java

```diff
@@ -58,30 +61,29 @@ public SpringDocConfiguration(SpringDocConfigProperties springDocConfigPropertie
      * @return a web security customizer with security settings for SpringDoc
      */
     @Bean
+    @Qualifier("springDocWebSecurityCustomizer")
```

Comment:
I would prefer just naming the bean this. The name serves as a qualifier in the abscence of a qualifier. If something were actually using the qualifier then I would go that route.

Reply:
I used `@Bean("springDocWebSecurityCustomizer")`